src/resolvers/bugs.js

```diff
@@ -14,6 +17,7 @@ export default {
       const query = {
         ...searchToQuery(search),
         include_fields: fieldsToIncludeFields(listFields(info)),
+        ...searchToQuery({ apiKey }),
```

Comment:
It seems that `searchToQuery` is spread twice here. Can we call it once instead by merging `search` and `{ apiKey }`?

Reply:
Sorry, didn't see that it was used right above. Should be fixed now.